core-feature-pack/common/src/main/resources/content/bin/installation-manager.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+# This script launches the operation to apply a candidate server installation to update or revert.
+# The server JVM writes the required values into the installation-manager.properties file by using InstMgrCandidateStatus.java
+if [ x"${INST_MGR_SCRIPT_DEBUG}" == "xtrue" ]; then
+  set -x
+fi
+
+INSTALLATION_HOME="${1}"
```

Comment:
My understanding is that this `installation-manager.sh` script should not be used directly by the users. If this is true, can we add some note about it to comments to the beginnig of the file and maybe inform the users about it here? If the users starts the script manually, the output is a little confusing (same for `.bat` alternative of this script):
```
$ ./installation-manager.sh 
INFO: Installation Manager properties file not found at /bin/installation-manager.properties.
$ ./installation-manager.sh --help
INFO: Installation Manager properties file not found at --help/bin/installation-manager.properties.
$ 
```